src/main/resources/db/migration/V0.5.15__create-inventory-tables.sql

```diff
@@ -0,0 +1,193 @@
+    program_id UUID,
+    type text,
+    active boolean DEFAULT true,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_type ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_type ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE inventory_relationship (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    inventory_relationship_type_id UUID NOT NULL,
+    relation_source_id UUID,
+    relation_target_id UUID,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_relationship ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_relationship ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE inventory_relationship_type (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    type text,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_relationship_type ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_relationship_type ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE inventory_source (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    inventory_source_type_id UUID NOT NULL,
+    relation_source_id UUID,
+    relation_target_id UUID,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_source ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_source ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE inventory_source_type (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    type text,
+    description text,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_source_type ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_source_type ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+CREATE TABLE inventory_unit_type (
+    like base_entity INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES,
+    program_id UUID,
+    type text,
+    like base_edit_track_entity INCLUDING ALL
+);
+
+ALTER TABLE inventory_unit_type ADD FOREIGN KEY (created_by) REFERENCES bi_user (id);
+ALTER TABLE inventory_unit_type ADD FOREIGN KEY (updated_by) REFERENCES bi_user (id);
+
+ALTER TABLE entity ADD FOREIGN KEY (entity_cross_id) REFERENCES entity_cross (id);
+ALTER TABLE entity ADD FOREIGN KEY (status_by) REFERENCES bi_user (id);
+
+ALTER TABLE entity_relationship ADD FOREIGN KEY (relation_source_id) REFERENCES entity (id);
+ALTER TABLE entity_relationship ADD FOREIGN KEY (relation_target_id) REFERENCES entity (id);
+
+ALTER TABLE entity_cross ADD FOREIGN KEY (mother_entity) REFERENCES entity (id);
+ALTER TABLE entity_cross ADD FOREIGN KEY (father_entity) REFERENCES entity (id);
```

Comment:
The table column is missing `_id` for both of these on the left side of the reference

Reply:
Fixed and pushed changes.